common/database_schema.h

```diff
@@ -148,6 +148,7 @@ namespace DatabaseSchema {
 			"guild_members",
 			"guild_ranks",
 			"guild_relations",
+			"guild_permissions",
```

Comment:
Should be right after `guild_members` for alphabetical order.

Reply:
:-)  Updated.